apps/web/src/pages/integrations/components/IntegrationInput.tsx

```diff
@@ -67,6 +67,23 @@ export function IntegrationInput({
     );
   }
 
+  if (credential.type === 'dropdown') {
+    return (
+      <Select
+        {...field}
+        label={credential.displayName}
+        required={credential.required}
+        description={credential.description}
+        data-test-id={credential.key}
+        error={errors[credential.key]?.message}
+        data={credential.dropdown?.map((item) => ({
+          label: item.name,
+          value: item.value,
+        }))}
+      />
+    );
+  }
```

Comment:
I made generic way to render dropdown

![screenshot](https://github.com/novuhq/novu/assets/43048524/7198fb8a-2c3e-4384-8113-816102941eab)